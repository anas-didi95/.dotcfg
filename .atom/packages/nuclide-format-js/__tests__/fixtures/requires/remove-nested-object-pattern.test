var {one: {two, four: [three]}} = require('foo');

one(four);
