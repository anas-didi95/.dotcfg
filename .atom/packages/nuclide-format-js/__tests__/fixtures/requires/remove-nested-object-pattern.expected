const four = require('four');
const one = require('one');

one(four);
